### ISSUE TITLE
TestContainers Mongo does not exist

### ISSUE THREAD

**Issue by hantsy** [testcontainers/testcontainers-java#2693]
Try to add Mongo test containers to my Spring tests, I added dependency as described in [the official docs](https://www.testcontainers.org/modules/databases/mongodb/).

```xml
<dependency>
    <groupId>org.testcontainers</groupId>
    <artifactId>mongodb</artifactId>
    <version>1.14.1</version>
    <scope>test</scope>
</dependency>
```

It does not exist, check: https://mvnrepository.com/artifact/org.testcontainers

**Comment by bsideup** [comment]
@hantsy good catch. It was just merged, but not released yet, while the docs render the latest state (basically, a SNAPSHOT).

@rnorth we need to think how to prevent it in future (maybe we can add `@since` tag to pages in mkdocs and include them only when version >= or something)

**Comment by jameskleeh** [comment]
This has just caused me 45 mins of hair pulling

**Comment by AElMehdi** [comment]
Any idea when is it going to come out @bsideup?
Just curious @hantsy; what are you using for now?
Thanks

**Comment by bsideup** [comment]
@AElMehdi released in 1.14.2.

**Comment by hantsy** [comment]
@AElMehdi [Updated to 1.14.2](https://github.com/hantsy/angular-spring-reactive-sample/blob/master/server/pom.xml#L23), check my example [here](https://github.com/hantsy/angular-spring-reactive-sample/blob/master/server/src/test/java/com/example/demo/PostRepositoryTest.java).
